tldw_Server_API/app/api/v1/endpoints/media/process_videos.py

```
@@ -262,6 +267,77 @@ async def process_videos_endpoint(
     except Exception as debug_err:  # pragma: no cover - defensive logging
         logger.error(f"Error during debug logging: {debug_err}")
 
+    # Optional template/hierarchical re-chunking of video transcripts (best-effort).
+    try:
+        if form_data.perform_chunking:
+            chunk_options_dict = prepare_chunking_options_dict(form_data)
+            try:
+                TemplateClassifier = getattr(media_mod, "TemplateClassifier", None)
+            except Exception:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)